src/api/coin.ts

```diff
@@ -32,8 +32,8 @@ export class Coin {
     recipient: HexInput;
     amount: AnyNumber;
     coinType?: MoveStructType;
-    options?: GenerateTransactionOptions;
-  }): Promise<SingleSignerTransaction> {
+    options?: InputGenerateTransactionOptions;
```

Comment:
I think in most places we use `Input` as profix (i.e at the end of the name) - we should go with one format

Reply:
ok. lets consolidate all types then